test_rclcpp/test/test_n_nodes.py.in

```diff
@@ -0,0 +1,45 @@
+# generated from test_rclcpp/test/test_n_nodes.py.in
+
+import os
+
+from launch.legacy import LaunchDescriptor
+from launch.legacy.exit_handler import primary_exit_handler
+from launch.legacy.launcher import DefaultLauncher
+
+
+def test_@TEST_NUM_NODES@_nodes():
+    ld = LaunchDescriptor()
+
+    for i in range(0, @TEST_NUM_NODES@):
+        cmd = ['@TEST_EXECUTABLE1@']
+        env = None
```

Comment:
The `env` can be initialized once outside the loop and also be reused for the check process.

Reply:
Yep.  I also realized it can be reused for the "check" node too, so I'm just using a single `env` at the top in 5a9c83ed1c9fed1b567bcf8a431a0a849e384aca